dags/curated_quanumchartmaxx.py

```diff
@@ -74,35 +74,45 @@
 )
 
 
-def generate_spark_arguments(destination: str, steps: str = "default", etl_version: str = "v2",
-                             subzone: str = "curated") -> List[str]:
+def generate_spark_arguments(destination: str, specific_main_method: bool, pass_date: bool,
```

Comment:
peut-être `include_date` ? j'ai vu pass_date j'ai pas compris au départ

Reply:
on dirait que maintenant je me suis habitué au pass_date nvm

Reply:
Hahahaha ouais c'était dans le sens pass date to main mais j'avoue on dirait que ça peut vouloir dire skip aussi...